src/SFML/Window/JoystickImpl.hpp

```diff
@@ -64,12 +64,12 @@ struct JoystickState
     JoystickState()
     {
         connected = false;
-        std::fill(axes, axes + Joystick::AxisCount, 0.f);
+        std::fill(std::begin(axes), std::end(axes), 0.f);
         std::fill(buttons, buttons + Joystick::ButtonCount, false);
```

Comment:
If we use std::array, we won't need this

Reply:
same